src/utils/base.js

```diff
@@ -68,7 +77,8 @@ export const getStyles = function getStyles() {
     textAlign,
     bgColor,
     bgImage,
-    bgDarken,
+    bgOverlayColour,
+    bgOverlayIntensity,
```

Comment:
I wouldn't call this prop "intensity" as that would refer to the saturation of the colour: https://www.uwgb.edu/heuerc/2D/ColorTerms.html

Instead I would just call this "opacity" or "opacify" or "transparentize" depending on what it'll do.

Reply:
Its bgTransparentize now ✨